Firestore/tests/Unit/ValueMapperTest.php

```diff
@@ -192,9 +191,11 @@ public function decodedValues()
         $blobValue = 'hello world';
         $blob = new Blob($blobValue);
 
-        $now = (string) time();
-        $nanos = 10;
-        $timestamp = new Timestamp(\DateTimeImmutable::createFromFormat('U', (string) $now), $nanos);
+        $datetime = \DateTimeImmutable::createFromMutable(\DateTime::createFromFormat('U.u', microtime(true)));
```

Comment:
I think this can just be `$datetime = \DateTimeImmutable::createFromFormat([..])` to fix the PHP 5.5 error in CI.